.circleci/config.yml

```diff
@@ -54,6 +54,9 @@ unit_tests: &unit_tests
         command: npm run check-install
     - run:
         name: Run system tests (skipped for PRs)
+        environment:
+          GCLOUD_PROJECT: node-team-test
```

Comment:
Could we please switch this to `long-door-651`?

Reply:
Sure. Could you give me permissions to the GCP project so I can check in a new key file as well?

Reply:
done!